contrib/html/html.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import defineFunction, {ordargument} from "../../src/defineFunction";
+import buildCommon from "../../src/buildCommon";
+import mathMLTree from "../../src/mathMLTree";
+import domTree from "../../src/domTree";
+import {defineGroupParser, newArgument} from "../../src/Parser";
+import ParseNode from "../../src/ParseNode";
+
+import {buildExpression as htmlBuildExpression} from "../../src/buildHTML";
+import {buildExpression as mmlBuildExpression} from "../../src/buildMathML";
+
+import type {ParsedArg} from "../../src/Parser";
+
+defineGroupParser("string", function(optional: boolean): ?ParsedArg {
+    const res = this.parseStringGroupWithBalancedBraces("string", optional);
+    if (!res) {
+        return null;
+    }
+    return newArgument(new ParseNode("string", res.text, "text"), res);
+});
+
+// $FlowFixMe
```

Comment:
What's the error?

Reply:
@kevinbarabash `props.argTypes` expects an array of `ArgType`, so it doesn't recognize custom `string` group type. I couldn't find a way to extend `ArgType` from `defineGroupParser`.

Reply:
I think it's fine to just change it in types.js.  Eventually though, we may say that
```
type ArgType = string;
```
and add other programmatic checks as necessary.